applications/nrf_desktop/src/events/hid_event.h

```diff
@@ -52,15 +53,16 @@ struct hid_mouse_event {
 EVENT_TYPE_DECLARE(hid_mouse_event);
 
 
-/** @brief Consumer control report data event. */
-struct hid_consumer_ctrl_event {
+/** @brief System/consumer control report data event. */
+struct hid_ctrl_event {
 	struct event_header header; /**< Event header. */
 
+	enum in_report report_type; /**< Type of HID control (system/consumer). */
```

Comment:
I know that both these control reports are in a way similar and pass 1 usage.
So far a report type was distinguished by event type. This change uses different method of passing report type.
This is why I felt against this change initially. I leave this is is as we plan to rework a way reports are passed around and so it is not very important to be super consistent here (in cost of additional work we would have to put to change it).

Reply:
Initially I was thinking about using multiple events (one per report).
Since these events would be used in the application in nearly the same way, I decided to use single event to avoid duplication. Other reports could be handled in similar way. It is also closer to the changes we are going to introduce in rework.